Commands.md

```diff
@@ -928,6 +929,58 @@ For example, running `git show-tree` will display:
 
 Be free to try it for yourself!
 
+
+## git stamp
+
+Stamp the last commit message
+
+Commit message is
+
+```bash
+Fix timezone bug
+```
+
+Reference the issues numbers from your bug tracker
+
+```bash
+$ git stamp Issue FOO-123
+$ git stamp Issue FOO-456 \#close
+
+Fix timezone bug
+
+Issue FOO-123
+
+Issue FOO-456 #close
+```
+
+Link to its review page
+
+```bash
+$ git stamp Review https://reviews.foo.org/r/4567/
+
+Fix timezone bug
+
+Issue FOO-123
+
+Issue FOO-456 #close
+
+Review https://reviews.foo.org/r/4567/
+```
+
+Replace previous issues with a new one  
+(Note that the identifier is case insensitive)
+
+```bash
+$ git stamp --replace issue BAR-123
+
+Fix timezone bug
+
+Review https://reviews.foo.org/r/4567/
+
+issue BAR-123
+```
+
+
```

Comment:
You mean showing the exact result of the commands as it they were executed?
Meaning with the commit hash, author name and date as well